lib/rules/no-implicit-globals.js

```diff
@@ -77,6 +77,13 @@ module.exports = {
                         return;
                     }
 
+                    // Variables exported by "exported" block comments
+                    const isExportedEslintVariables = variable.eslintExported;
+
+                    if (isExportedEslintVariables) {
```

Comment:
We don’t need a variable here. 

```suggestion
                    if (variable.eslintExported) {
```

Reply:
done! [42ac8ae](https://github.com/eslint/eslint/pull/16343/commits/42ac8ae703199932d0068975e7205a28efeb0ea4)